patterns/design-patterns-behavioral/src/main/java/com/baeldung/nulls/UsingOptional.java

```diff
@@ -11,6 +15,18 @@
         return Optional.ofNullable(response);
     }
 
+    public String findFirst(){
+
+        String firstElement = getList().stream()
+                .findFirst()
+                .orElseThrow(IllegalArgumentException::new);
+        return firstElement;
```

Comment:
this may be even easier to read if you put the `return` statement on line 20 rather than capture a local variable.